### DISCUSSION TITLE
Aiuto classi alternate

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#603] [Richieste di aiuto]
Buonasera ,c'è un problema:ho cercato di creare le classi alternate seguendo le vostre guide , ma nel momento in cui ad esempio dalla classe 3CL creo due classi con gruppi CIN (cinese) e TED (tedesco) e attribuisco le rispettive cattedre ovvero il cinese per (3CL-CIN e  il tedesco 3CL-TED con i rispettivi docenti  delle seguenti materie di insegnamento ,questi entrano nel registro ,ma nel momento in cui devono fare l'appello ,inserire il voto o qualsiasi altra cosa non trovano l'elenco degli alunni ,come mai ?Dove sbaglio

**Comment by trinko** [comment]
Come hai impostato gli alunni? 
Questi devono essere attribuiti ai gruppi classi: 3CL-CIN o 3CL-TED, mai su 3CL.

**Comment by sempliciniluigi89** [comment]
Non sto capendo,mi scusi
 ,potrebbe essere più chiaro?

Il giorno lun 11 dic 2023 alle ore 23:20 Antonello Dessì <
***@***.***> ha scritto:

**Comment by trinko** [comment]
Bisogna indicare quali alunni fanno parte del gruppo 3CL-CIN o 3CL-TED. Semplicemente si fa impostando la classe dell'alunno su 3CL-CIN o su 3CL-TED.

**Comment by sempliciniluigi89** [comment]
Come posso fare se nella mia classe di indirizzo linguistico una volta alla
settimana c'è la compresenza con il  madrelingua che compila il registro
,ma anche il docente di lingua vuole compilare il registro per conto suo
senza interferire su quello che inserisce il madrelingua ,come si potrebbe
fare in modo che le firme,le compilazioni siano differenti ,ma nella stessa
ora ?

Il giorno lun 11 dic 2023 alle ore 23:36 Antonello Dessì <
***@***.***> ha scritto:

**Comment by trinko** [comment]
Se entrambi fanno lezione a tutti gli alunni non si possono usare i gruppi classe.

La situazione del madrelingua mi pare del tutto analoga a quella della compresenza con l'insegnante di laboratorio che noi abbiamo nel nostro Istituto.
Per cui si può fare nel medesimo modo: per la stessa classe e materia si definisce una cattedra per il docente di lingua e una per il madrelingua, quest'ultima impostata come ITP.
A questo punto entrambi possono firmare la stessa ora: anche se la descrizione della lezione svolta è  condivisa, si può stabilire che il docente di lingua usi solo il campo ARGOMENTI e il madrelingua usi le ATTIVITA'. 
Per i voti, entrambi possono inserire voti in modo indipendente. Per una questione di maggiore chiarezza, è meglio separare i voti per tipologia: il madrelingua usarà il voto per la PRATICA, il docente quello orale/scritto.
Infine, per gli scrutini la proposta di  vota sarà unica, perché la materia è una sola.

**Comment by sempliciniluigi89** [comment]
Va bene,grazie ,lo chiedevo dal momento che nel registro Argo
,Nuvola,Classeviva questo è  possibile.

Il Mar 12 Dic 2023, 00:26 Antonello Dessì ***@***.***> ha
scritto:
